test_harness/run.py

```diff
@@ -1,37 +1,164 @@
 """Run tests through the Test Runners."""
-
+from typing import Optional, List, Dict
 from collections import defaultdict
+import httpx
 import json
 import logging
 import time
 from tqdm import tqdm
 import traceback
-from typing import Dict, List
 
-from ARS_Test_Runner.semantic_test import run_semantic_test as run_ars_test
-from benchmarks_runner import run_benchmarks
+from translator_testing_model.datamodel.pydanticmodel import (
+    TestAsset,
+    TestCase,
+    TestEnvEnum,
+    ComponentEnum
+)
 
-from translator_testing_model.datamodel.pydanticmodel import TestCase
+from ARS_Test_Runner.semantic_test import run_semantic_test as run_ars_test
+from standards_validation_test import run_standards_validation_tests
+from one_hop_test import run_one_hop_tests
+# from benchmarks_runner import run_benchmarks
 
 from .reporter import Reporter
 from .slacker import Slacker
 
 
+def get_tag(result):
```

Comment:
I think we're now firmly in the realm of creating a utils file with all these small helper functions.